server/service/osquery_utils/queries.go

```diff
@@ -1325,10 +1349,12 @@ func GetDetailQueries(
 		generatedMap["software_macos"] = softwareMacOS
 		generatedMap["software_linux"] = softwareLinux
 		generatedMap["software_windows"] = softwareWindows
+		generatedMap["software_chrome"] = softwareChrome
 	}
 
 	if features != nil && features.EnableHostUsers {
 		generatedMap["users"] = usersQuery
+		generatedMap["users"] = usersQueryChrome
```

Comment:
This overwrites the users query and should really be something like

```suggestion
		generatedMap["users_chrome"] = usersQueryChrome
```

This causes test failures as well.

Reply:
Thank you unit tests! (bis)